app/serializers/meeting_serializer.rb

```diff
@@ -1,5 +1,12 @@
 class MeetingSerializer
 include JSONAPI::Serializer
-  
-  attributes :date, :start_time, :end_time, :is_accepted, :purpose, :partner_id, :user_id
+  attributes :date, :start_time, :end_time, :is_accepted, :purpose, :partner_id
+
+  attribute :start_time do |meeting|
+    meeting.start_time.strftime("%I:%M %p")
+  end
+
+  attribute :end_time do |meeting|
```

Comment:
Ah I see, this is where you had to manually convert the time to be the actual time and not the wrong date; nice catch!